homework-submission/week1/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Welcome to React World</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+  <body>
+    <div id="thisIsWhereReactIsInjected">
+      <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
```

Comment:
exactly! it's as it sound: `thisIsWhereReactIsInjected`

And that's why you should avoid typing your script tags here, because they will be removed in some point instead of re-using them as many times as you need!

Looking to the question again:
> A simple HTML file, with one `<div>` that has an id of `thisIsWhereReactIsInjected` and the following scripts included **_in_** the `<body>`